streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/entity/User.java

```diff
@@ -102,4 +102,9 @@ public class User implements Serializable {
      */
     private Long teamId;
 
+    public void dataMasking() {
+        this.setPassword("******");
```

Comment:
hi， @wolfboys  Thank you for your contribution。Strings with special meanings are recommended to be extracted as public variables，do you think？

Reply:
> hi， @wolfboys Thank you for your contribution。Strings with special meanings are recommended to be extracted as public variables，do you think？

Great suggestion, I agree with you.  I'll fix it later